src/main/java/frc/robot/subsystems/HopperController.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANError;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import io.github.oblarg.oblog.annotations.Config;
+
+public class HopperController {
+
+    private final int LEFT_CAN_ID = 9;
+    private final int RIGHT_CAN_ID = 10;
+
+    private CANSparkMax leftController;
+    private CANSparkMax rightController;
+
+    private double speed = .3;
+
+    public HopperController() {
+        this.leftController = new CANSparkMax(LEFT_CAN_ID, MotorType.kBrushless);
+        this.rightController = new CANSparkMax(RIGHT_CAN_ID, MotorType.kBrushless);
+        this.setMasterSlave();
+    }
+
+    public void setMasterSlave() {
+        final CANError slave = this.rightController.follow(this.leftController);
```

Comment:
I wasn't in on Sunday, so maybe this changed, but I thought that the two sides of the hopper are supposed to spin opposite of one another.

Reply:
The motors are mirrored, so they are flipped :)